internal/distro/rhel8/azure.go

```diff
@@ -0,0 +1,573 @@
+			"aic94xx-firmware",
+			"alsa-firmware",
+			"alsa-lib",
+			"alsa-sof-firmware",
+			"alsa-tools-firmware",
+			"dracut-config-rescue",
+			"ivtv-firmware",
+			"iwl1000-firmware",
+			"iwl100-firmware",
+			"iwl105-firmware",
+			"iwl135-firmware",
+			"iwl2000-firmware",
+			"iwl2030-firmware",
+			"iwl3160-firmware",
+			"iwl3945-firmware",
+			"iwl4965-firmware",
+			"iwl5000-firmware",
+			"iwl5150-firmware",
+			"iwl6000-firmware",
+			"iwl6000g2a-firmware",
+			"iwl6000g2b-firmware",
+			"iwl6050-firmware",
+			"iwl7260-firmware",
+			"libertas-sd8686-firmware",
+			"libertas-sd8787-firmware",
+			"libertas-usb8388-firmware",
+			"glibc-all-langpacks",
+			"biosdevname",
+			"cockpit-podman",
+			"bolt",
+			"buildah",
+			"containernetworking-plugins",
+			"dnf-plugin-spacewalk",
+			"iprutils",
+			"plymouth",
+			"podman",
+			"python3-dnf-plugin-spacewalk",
+			"python3-rhnlib",
+			"python3-hwdata",
+			"NetworkManager-config-server",
+			"rhn-client-tools",
+			"rhn-setup",
+			"rhnsd",
+			"rhn-check",
+			"rhnlib",
+			"usb_modeswitch",
+		},
+	}.Append(bootPackageSet(t)).Append(distroSpecificPackageSet(t))
+
+	if t.arch.distro.isRHEL() {
+		ps.Append(rpmmd.PackageSet{
+			Include: []string{
+				"insights-client",
+				"rhc",
+			},
+		})
+	}
+
+	return ps
+}
+
+// Azure BYOS image package set
+func azurePackageSet(t *imageType) rpmmd.PackageSet {
+	return azureCommonPackageSet(t)
+}
+
+// Azure RHUI image package set
+func azureRhuiPackageSet(t *imageType) rpmmd.PackageSet {
+	return rpmmd.PackageSet{
+		Include: []string{
+			"rhui-azure-rhel8",
+		},
+	}.Append(azureCommonPackageSet(t))
+}
+
+// PARTITION TABLES
+
+var azureRhuiBasePartitionTables = distro.BasePartitionTableMap{
+	distro.X86_64ArchName: disk.PartitionTable{
+		UUID: "D209C89E-EA5E-4FBD-B161-B461CCE297E0",
+		Type: "gpt",
+		Size: 68719476736,
+		Partitions: []disk.Partition{
+			{
+				Size: 524288000,
+				Type: disk.EFISystemPartitionGUID,
+				UUID: disk.EFISystemPartitionUUID,
+				Payload: &disk.Filesystem{
+					Type:         "vfat",
+					UUID:         disk.EFIFilesystemUUID,
+					Mountpoint:   "/boot/efi",
+					FSTabOptions: "defaults,uid=0,gid=0,umask=077,shortname=winnt",
+					FSTabFreq:    0,
+					FSTabPassNo:  2,
+				},
+			},
+			{
+				Size: 524288000,
+				Type: disk.FilesystemDataGUID,
+				UUID: disk.FilesystemDataUUID,
+				Payload: &disk.Filesystem{
+					Type:         "xfs",
+					Mountpoint:   "/boot",
+					FSTabOptions: "defaults",
+					FSTabFreq:    0,
+					FSTabPassNo:  0,
+				},
+			},
+			{
+				Size:     2097152,
+				Bootable: true,
+				Type:     disk.BIOSBootPartitionGUID,
+				UUID:     disk.BIOSBootPartitionUUID,
+			},
+			{
+				Type: disk.LVMPartitionGUID,
+				UUID: disk.RootPartitionUUID,
+				Payload: &disk.LVMVolumeGroup{
+					Name:        "rootvg",
+					Description: "built with lvm2 and osbuild",
+					LogicalVolumes: []disk.LVMLogicalVolume{
+						{
+							Size: 1 * 1024 * 1024 * 1024,
```

Comment:
Super duper nit that can definitely be done in a follow up: you introduced `common.GibiByte` but didn't use it here ;)

Reply:
You are right 🙂 My idea was to change all partition tables in all distros to use it in a follow up. Happy to do it as part of this PR, I just wanted to minimize the risk of conflicts.

Reply:
Follow up is totally fine, even preferred.